src/main/scala/edu/ie3/simona/model/grid/RefSystem.scala

```diff
@@ -121,10 +126,13 @@ final case class RefSystem private (
     * @param qInPu
     *   referenced active power value in p.u.
     * @return
-    *   unreferenced active power value in Var
+    *   unreferenced active power value in VAr
```

Comment:
I'm not 100% sure, but I think units that aren't dedicated to Persons are all written lowercase. So i would suggest var. (Just for the pedantic part in everyone of us :) )

Reply:
I think you're right that ["the correct symbol is lower-case 'var', although the spellings 'Var' and 'VAr' are commonly seen"](https://en.wikipedia.org/wiki/Volt-ampere#Reactive).

However, we're using VAr in [PSU](https://github.com/ie3-institute/PowerSystemUtils/blob/24de7ea349f19eee0fd45395aeac5e8ddee11a6d/src/main/java/edu/ie3/util/quantities/PowerSystemUnits.java#L116), so that's why I'm using it here as well.

Reply:
Well, this doesn't change my opinion on this :) but I'm fine with both since it's clear what es meant here and thats the important thing. I just got on this since there is the proposed change from Var in VAr...

I checked DIN 1301 Teil 1 just for the sake of it: Unit name:  Var and Unit symbol: var.  I proposed this to [PSU](https://github.com/ie3-institute/PowerSystemUtils/pull/279)

Reply:
Great! Regarding this PR, I reverted the naming to the correct unit name `Var`.